keps/sig-network/2595-expanded-dns-config/README.md

```diff
@@ -103,6 +103,14 @@ DNS search path list to an arbitrary number
 
 ### Notes/Constraints/Caveats (Optional)
 
+Some container runtimes of older versions have their own restrictions on the
+number of DNS search paths. For the container runtimes which are older than
+- Containerd v1.6
```

Comment:
@mikebrow can you please give us some insights on containerd timelines?
The goal is to safely enabled this feature

Reply:
https://github.com/containerd/containerd/blob/main/RELEASES.md#support-horizon

>  containerd v1.5, End of life: max(May 3, 2022, release of 1.6.0 + 6 months)

Reply:
UPDATED: 

https://github.com/containerd/containerd/blob/main/RELEASES.md#support-horizon

> Release | Status | Start | End of Life
> -- | -- | -- | --
> ...
> 1.5 | Active | May 3, 2021 | October 28, 2022